OPConsultation-output.json

```diff
@@ -0,0 +1,2874 @@
+        "requester": {
+          "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+          "type": "Practitioner"
+        },
+        "dosageInstruction": [
+          {
+            "doseAndRate": [
+              {
+                "doseQuantity": {
+                  "value": 500,
+                  "code": "mg",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "mg"
+                },
+                "rateQuantity": {
+                  "value": 3,
+                  "code": "{1/d}",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "1/d"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:dc0d79a6-2611-4917-9d34-e6a98acede05",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "dc0d79a6-2611-4917-9d34-e6a98acede05",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
+            {
+              "code": "367336001",
+              "system": "http://snomed.info/sct",
+              "display": "Chemotherapy"
+            }
+          ],
+          "text": "Chemotherapy"
+        },
+        "subject": {
+          "reference": "urn:uuid:c4d052b5-2d9f-4ebf-b617-764efffa08de",
+          "type": "Patient"
+        },
+        "authoredOn": "2024-08-06T12:18:11+05:30",
+        "requester": {
+          "reference": "urn:uuid:41295111-04f9-4b83-b186-ef2975db1c7e",
+          "type": "Practitioner"
+        },
+        "dosageInstruction": [
+          {
+            "doseAndRate": [
+              {
+                "doseQuantity": {
+                  "value": 500,
+                  "code": "mg",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "mg"
+                },
+                "rateQuantity": {
+                  "value": 3,
+                  "code": "{1/d}",
+                  "system": "http://unitsofmeasure.org",
+                  "unit": "1/d"
+                }
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "fullUrl": "urn:uuid:b0267f24-a264-43d2-8813-ef8de560269f",
+      "resource": {
+        "resourceType": "MedicationRequest",
+        "id": "b0267f24-a264-43d2-8813-ef8de560269f",
+        "meta": {
+          "versionId": "0",
+          "lastUpdated": "2024-08-06T12:18:11.223+05:30",
+          "profile": [
+            "https://nrces.in/ndhm/fhir/r4/StructureDefinition/MedicationRequest"
+          ]
+        },
+        "status": "active",
+        "intent": "plan",
+        "medicationCodeableConcept": {
+          "coding": [
```

Comment:
really???? 385388004 is concept for indicating lymphoma stage! How is this a medication??